iceoryx_hoofs/include/iceoryx_hoofs/internal/posix_wrapper/shared_memory_object/shared_memory.hpp

```diff
@@ -69,7 +70,7 @@ class SharedMemory
     ~SharedMemory() noexcept;
 
     /// @brief returns the file handle of the shared memory
-    int32_t getHandle() const noexcept;
+    int32_t get_file_handle() const noexcept;
```

Comment:
Similar to what I wrote in the last PR. It is a bit weird to have two different naming conventions in the same class but I guess it is the same convention as in `FileManagementInterface`. Therefore there is already the issue with different naming conventions and the user does not know anyway if a method comes from `SharedMemory` or  `FileManagementInterface`. Ideally, all the classes in hoofs would be converted to a common naming convention ... something for our future selves